From: Yaroslav Halchenko <debian@onerussian.com>
Subject: Downgrade requirement for future

I believe that 0.16.0 was demanded for compatibility with python 3.6.  All
tests seems to function correctly with 0.15.2

Last-Update: 2017-05-23

--- a/requirements.txt
+++ b/requirements.txt
@@ -4,7 +4,7 @@ networkx>=1.9
 traits>=4.6
 python-dateutil>=2.2
 nibabel>=2.1.0
-future>=0.16.0
+future>=0.15.2
 simplejson>=3.8.0
 prov>=1.5.2
 neurdflib
--- a/nipype/info.py
+++ b/nipype/info.py
@@ -109,7 +109,7 @@ SCIPY_MIN_VERSION = '0.14'
 TRAITS_MIN_VERSION = '4.6'
 DATEUTIL_MIN_VERSION = '2.2'
 PYTEST_MIN_VERSION = '3.6'
-FUTURE_MIN_VERSION = '0.16.0'
+FUTURE_MIN_VERSION = '0.15.2'
 SIMPLEJSON_MIN_VERSION = '3.8.0'
 PROV_VERSION = '1.5.2'
 CLICK_MIN_VERSION = '6.6.0'
